module/library/categories/helper.js

```diff
@@ -593,10 +608,37 @@ module.exports = class LibraryCategoriesHelper {
 	static create(categoryData, files, userDetails) {
 		return new Promise(async (resolve, reject) => {
 			try {
+				const tenantId = userDetails.tenantAndOrgInfo.tenantId
+				const orgId = userDetails.tenantAndOrgInfo.orgId
+
+				// Check if organization extension exists for the loggedin user
+				let orgExtension
+				orgExtension = await orgExtensionQueries.orgExtenDocuments({
+					tenantId,
+					orgId: orgId[0],
+				})
+				// Create default org-extension policy if not found
+				if (!orgExtension || orgExtension.length === 0) {
+					orgExtension = await orgExtensionQueries.create({
+						tenantId,
+						orgId: orgId[0],
+						...CONSTANTS.common.DEFAULT_ORG_EXTENSION_POLICIES,
```

Comment:
@Prajwal17Tunerlabs do we have to explicitly pass this? we already have default values mentioned in model I suppose. that is not enough?